guake/test_terminal.py

```diff
@@ -43,7 +38,7 @@
 def main():
     terminal = Vte.Terminal()
 
-    terminal.spawn_sync(
+    terminal.fork_command_full(
```

Comment:
hum, vte 0.38 has replaced fork_command_full by spawn_sync, which version of VTE do you have?